README.md

```
@@ -1,19 +1,325 @@
-or Run by docker compose
+* **Login to your AWS account** through the [AWS Console](https://aws.amazon.com/console/).
+    
+* **Launch an EC2 instance**:
+    
+    * Navigate to the **EC2 Dashboard** and click on **Launch Instance**.
+        
+    * Choose **Ubuntu 24.04 LTS** as the operating system.
+        
+    * Select the **t2.micro** instance type (free-tier eligible).
+        
+    * Configure your **security groups** to allow **SSH (port 22)** access and also prepare ports **8080** (for Jenkins) and **8000** (for the web app).
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)